docs/dev-guide/integration/rest-api/tasks.rst

```diff
@@ -120,6 +138,24 @@ All currently running and waiting tasks may be listed. This returns an array of
 | :return:`array of` :ref:`task_report`
 
 
+Polling Task Group Progress
+---------------------------
+
+Poll a group of tasks for progress summary. Polling returns a :ref:`task_group_summary`
+
+| :method:`get`
+| :path:`/v2/task_groups/<task_group_id>/state_summary/`
+| :permission:`read`
+
+| :response_list:`_`
+
+* :response_code:`200, if the task is found`
+* :response_code:`404, if the task group id is not found`
+
+| :return:`a` :ref:`task_group_summary` representing the task queried
```

Comment:
s/representing  the task queried/ representing the summary state of queried task group id